src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,39 +1,56 @@
 package edu.kis.vh.nursery;
 
+import edu.kis.vh.nursery.stackable.IntArrayStack;
+import edu.kis.vh.nursery.stackable.IntLinkedList;
+import edu.kis.vh.nursery.stackable.IntStackableInterface;
+
 public class DefaultCountingOutRhymer { //All of the tests passed without problem
     //Checked how open Declaration works - it shows where selected method is used.
 
-    private IntArrayStack intArrayStack;
+    private IntStackableInterface intStackableInterface;
```

Comment:
"interface" suffix of the attribute name is wrong and does not show the meaning.